network/update-proxy-configs

```diff
@@ -155,6 +155,7 @@ fi
 if [ -e /etc/portage/make.conf ]; then
     update_conf /etc/portage/make.conf "http_proxy=\"$PROXY_ADDR\"
 https_proxy=\"$PROXY_ADDR\"
+no_proxy=\"${NO_PROXY:-127.0.0.1}\"
```

Comment:
I don't see it documented in make.conf(5) man page, do you have any source for this?